packages/main/src/Select.ts

```diff
@@ -1095,7 +1094,7 @@ class Select extends UI5Element implements IFormElement {
 	}
 
 	get _filteredItems() {
```

Comment:
I believe we can now remove the _filteredItems getter and use this.options directly in these places:

<img width="807" alt="Screenshot 2024-04-03 at 8 52 02" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/d47b2184-8853-4cac-947b-0896b6a8923d">